gepetuto/main.py

```diff
@@ -97,26 +105,36 @@ def retrieve_python_interpreter():
             return sys.executable
 
 
-def get_file_list(tp_id, file):
+def get_file_list(args):
     """Get the list of files we use action on."""
-    file = [Path(f) for f in file]
-    file_list = []
-    if tp_id == []:
+    if args.tp_id == []:
         tp_id = get_tp_id()
+    else:
+        tp_id = args.tp_id
+    if args.file == []:
+        file = []
+    else:
+        file = [Path(f) for f in args.file]
```

Comment:
```suggestion
    file = [Path(f) for f in args.file]
```
if args.file is empty, the behaviour is the same